Forge Standard Library docs (#207)
foundry-rs/book: src/reference/forge-std/deal.md

## `deal`

### Signature

```solidity
function deal(address to, uint256 give) public;
```

```solidity
function deal(address token, address to, uint256 give) public;
```

```solidity
function deal(address token, address to, uint256 give, bool adjust) public;
```

### Description

> The same as [`deal`](../../cheatcodes/deal.md) cheatcode, but works for any ERC-20 token as well.

Reviewer: Some non-standard tokens may not work last I checked

```suggestion
A wrapper around the [`deal`](../../cheatcodes/deal.md) cheatcode that also works for most ERC-20 tokens.
```
